src/components/node-list/node-list-tree.js

```diff
@@ -100,6 +102,13 @@ const TreeListProvider = ({
     focusMode,
   ]);
 
+  useEffect(() => {
+    // Initial state contracts all modular pipelines
+    for (const id of modularPipelineIds) {
+      onToggleContracted(id, true);
+    }
```

Comment:
rather than looping through all the modular pipelines to contract one by one, it would be easier if we just set this in the default state when initialising the state>